support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.scss

```diff
@@ -89,32 +89,26 @@
     line-height: 22px;
   }
 
-  .confirmation {
+  .contribution-thank-you-block {
```

Comment:
👍 for the rename

Reply:
i'll put a card in the backlog for combing through the thank you pages more thoroughly. Each of these blocks on the thank you page has its own class but the same styles.